README.md

```diff
@@ -1653,6 +1683,8 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 **[`^        back to top        ^`](#)**
 
+A [search engine](https://en.wikipedia.org/wiki/Search_engine) is a software system designed to carry out [web searches](https://en.wikipedia.org/wiki/Web_query).
```

Comment:
I think the correct description would be that one (these are not only Web search engines):

```
A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an [information retrieval system](https://en.wikipedia.org/wiki/Information_retrieval) designed to help find information stored on a computer system. This includes [Web search engines](https://en.wikipedia.org/wiki/Web_search_engine).
```